src/routes/renderRoutes.js

```diff
@@ -30,7 +30,9 @@ routes.landingpage = async (req, res) => {
   if (!user.dbPassword) {
     return res.render("setDBpassword", { username: req.session.username });
   }
-  res.render("databases", { username: req.session.username });
+  res.render("welcome", {
+    username: req.session.username,
+  });
```

Comment:
Changing this means that now users can't go to databases page... I think we need alternative feature that allows users to see databases page. How about redirecting users to databases page when they click database buttons?